persistence/src/commonMain/db_user/com/wire/kalium/persistence/Clients.sq

```diff
@@ -28,3 +28,6 @@ SELECT * FROM Client;
 
 selectAllClientsByUserId:
 SELECT * FROM Client WHERE user_id = ?;
+
+selectAllClientsByConversation:
+SELECT * FROM Client WHERE user_id IN (SELECT user FROM Member WHERE conversation LIKE ('%' || :conversation_id || '%'));
```

Comment:
i think this will allow to use the full conversation id not just it's value
```
SELECT * FROM Client WHERE user_id IN (SELECT user FROM Member WHERE conversation = :conversationId );
```

Reply:
hmm, true, i copied it from `Member.sq`